test/test_fx.py

```diff
@@ -2260,5 +2229,114 @@ def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
 
 instantiate_device_type_tests(TestOperatorSignatures, globals())
 
+@skipIfNoTorchVision
+class TestVisionTracing(JitTestCase):
+    PROXY_ITERATED = (TraceError, r"Proxy object cannot be iterated")
+    UNTRACEABLE_MODELS = {
+        "fasterrcnn_resnet50_fpn": PROXY_ITERATED,
+        "fasterrcnn_mobilenet_v3_large_320_fpn": PROXY_ITERATED,
+        "fasterrcnn_mobilenet_v3_large_fpn": PROXY_ITERATED,
+        "maskrcnn_resnet50_fpn": PROXY_ITERATED,
+        "keypointrcnn_resnet50_fpn": PROXY_ITERATED,
+        "retinanet_resnet50_fpn": PROXY_ITERATED,
+    }
+
+    output_transform = {
+        "googlenet": lambda x: x.logits,
+        "inception_v3": lambda x: x.logits,
+        "fcn_resnet50": lambda x: x["out"],
+        "fcn_resnet101": lambda x: x["out"],
+        "deeplabv3_resnet50": lambda x: x["out"],
+        "deeplabv3_resnet101": lambda x: x["out"],
+        "deeplabv3_mobilenet_v3_large": lambda x: x["out"],
+        "lraspp_mobilenet_v3_large": lambda x: x["out"],
+        "fasterrcnn_resnet50_fpn": lambda x: x[1],
+        "fasterrcnn_mobilenet_v3_large_fpn": lambda x: x[1],
+        "fasterrcnn_mobilenet_v3_large_320_fpn": lambda x: x[1],
+        "maskrcnn_resnet50_fpn": lambda x: x[1],
+        "keypointrcnn_resnet50_fpn": lambda x: x[1],
+        "retinanet_resnet50_fpn": lambda x: x[1],
+    }
+
+    @classmethod
+    def generate_test_fn(cls, name, model, x):
+        def run_test(self):
+            model.eval()
+            if name in self.UNTRACEABLE_MODELS:
+                err, exc = self.UNTRACEABLE_MODELS[name]
+                with self.assertRaisesRegex(err, exc):
+                    graph = symbolic_trace(model)
+            else:
+                out_transform = self.output_transform.get(name, lambda x: x)
+                graph : torch.fx.GraphModule = symbolic_trace(model)
+                script = torch.jit.script(graph)
+                a = out_transform(model(x))
+                b = out_transform(graph(x))
+                c = out_transform(script(x))
+                self.assertEqual(a, b)
+                self.assertEqual(a, c)
+
+                quantizer = Quantizer(graph)
+                for _ in range(10):
+                    quantizer.observe((x,))
+                qgraph = quantizer.quantize()
+                qgraph.graph.lint()
+                qgraph_script = torch.jit.script(qgraph)
+                d = out_transform(qgraph(x))
+                e = out_transform(qgraph_script(x))
+                torch.testing.assert_allclose(a, d)
```

Comment:
I think you're going to have to increase the tolerance (atol or rtol kwargs) if you're comparing between an unquantized and a quantized model. It might be a better idea to just remove the quantization tests, that stuff should be covered in `test_quantize.py` anyway